app/models/reports/repository.rb

```diff
@@ -142,7 +142,6 @@ def warm_cache
         blood_sugar_measures_by_user
         monthly_registrations_by_user(diagnosis: :hypertension)
         monthly_registrations_by_user(diagnosis: :diabetes)
-        monthly_registrations_by_gender
```

Comment:
We're omitting this because it's only for DHIS2 once a month?

Reply:
Yes.